### ISSUE TITLE
center display: inline-block;

### ISSUE THREAD

**Issue by drcll** [dompdf/dompdf#1635]
Hello, I have problem with center inline-block headline with padding and border radius. Im trying something like this:

```
<style>
   .wrapper{
       display:block;
       text-align:center;
   }
   h1 {
      display:inline-block;
      padding: 10px 20px;
      border-radius: 24px;
      background-color: blue;
   }
</style>
<div class="wrapper">
    <h1>Some title text</h1>
</div>
```

Problem is, that my h1 element is not centered. Any suggestions? Thanks

**Comment by netzgestaltung** [comment]
did you inspect if there are other CSS rules applied to .wrapper or h1 ?

**Comment by drcll** [comment]
is there any opinion how to check if  css rules are applioed in PDF?

but yes, other rules are working

**Comment by Mellthas** [comment]
This is implemented via #2505 now.
